src/components/card/CardSection.tsx

```diff
@@ -56,14 +54,6 @@ type Props = CardSectionProps & asCardChildProps;
 class CardSection extends PureComponent<Props> {
   static displayName = 'Card.Section';
 
-  constructor(props: Props) {
-    super(props);
-
-    if (props.imageSource) {
-      LogService.deprecationWarn({component: 'CardSection', oldProp: 'imageSource', newProp: 'source'});
-    }
-  }
-
```

Comment:
Unfortunately, I think now we need to have the opposite warning.. 
It's been for a few months, no?